hr_family/models/hr_children.py

```diff
@@ -19,7 +19,8 @@
 #
```

Comment:
@GillesTephaneMeyomesse Thanks for the contribution, can you update with Small Headers..